Trimble.Modus.Components/Controls/TMModal/TMModal.cs

```diff
@@ -149,8 +150,6 @@ private static void OnFullWidthButtonChanged(BindableObject bindable, object old
                     foreach (TMButton button in modal._buttonContainer.Children)
                     {
                         button.HorizontalOptions = LayoutOptions.FillAndExpand;
-                        button.frame.HorizontalOptions = LayoutOptions.FillAndExpand;

```

Comment:
Are these intentional? @BAL4-R can you confirm?

Reply:
Yes the FillAndExpand sets all the buttons in modal to be fullwidth